builder/qemu/builder.go

```diff
@@ -88,6 +88,8 @@ type Config struct {
 	DiskSize        uint       `mapstructure:"disk_size"`
 	DiskCache       string     `mapstructure:"disk_cache"`
 	DiskDiscard     string     `mapstructure:"disk_discard"`
+	DiskShrink      bool       `mapstructure:"disk_shrink"`
```

Comment:
Shrink and Compress is separated features, compress slowdown writing. But shrink not, so i think this is two independent options.